app/blobs/templates.json

```diff
@@ -0,0 +1,23 @@
+[
+    { 
+        "imageURL": "images/barchart.png",
+        "templateName" : "Bar Chart",
+        "data" : {
+            "columns": [
+                ["data1", 30, 200, 100, 400, 150, 250],
+                ["data2", 50, 20, 10, 40, 15, 25]
+            ]
+        },
+        "dims" : 2,
+        "id" : 1
+
+    },
+    { 
+        "imageURL" : "images/linegraph.png",
+        "templateName" : "Line Graph",
+        "data" : {"test": 456},
+        "dims" : 2,
+        "id" : 2
```

Comment:
So I did this because React throws a warning if you don't provide keys for children as you pass them around.  It suggested unique numbers.

But sure, SHA or MD5 hashes would be fine, if we really want to be safe; though, I don't see what the attack could be here.